### ISSUE TITLE
Accessibility Issue - Contrast on sds-search button

### ISSUE THREAD

**Issue by brandydanner-gsa** [GSA/sam-design-system#619]
**Components Involved**
sds-search

**Expected Behavior**
Contrast on search button meets accessibility requirements

**Actual Behavior**
Contrast on search button does not meet accessibility requirements: "This SPAN contains text with a background color of #3f57a6 rgb(63, 87, 166) and foreground color of #162e51 rgb(22, 46, 81) that is less than 18 point in size; or bold text less than 14 point in size that has a luminosity contrast ratio of 2.024, which is below 4.5:1"

**Your Environment**
Angular Version: 10.2
@gsa-sam packages version: 10.0.12, sam-styles 0.0.102

**Additional context**

![image](https://user-images.githubusercontent.com/75684861/110347951-22132480-7fff-11eb-9cc9-5395b7f41905.png)

**Comment by brandydanner-gsa** [comment]
Updated to packages 10.0.15 and I confirmed this is no longer a finding from Access Assistant.  Closing this ticket.
